volunteerapp/templates/volunteerapp/index.html

```diff
@@ -6,7 +6,7 @@ <h1 class="project-font"><strong> Projects </strong> </h1>
 
 <ul>
   {% for project in projects %}
-  <li><a id="custom-fontspec" class ="project-font" href="#">{{ project.title }}</a></li>
+  <li><a class="custom-fontspec" class ="project-font" href="#">{{ project.title }}</a></li>
```

Comment:
This wont work, you need to have only one class attribute. it should look like: `class="project-font custom-fontspec"`